backend/app/api/endpoints/openapi_responses.py

```
@@ -446,109 +446,27 @@ def _close_db():
         dispatch_task = asyncio.create_task(
             execution_dispatcher.dispatch(execution_request, emitter=emitter)
         )
-        accumulated_content, _ = await emitter.collect()
+        accumulated_content, final_event = await emitter.collect()
+        dispatch_error = None
         try:
             await dispatch_task
-        except Exception:
-            pass
-        return accumulated_content
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)